README.md

```diff
@@ -116,10 +114,44 @@ The following prerequisites are required:
 
+## Deploy CoatRack with FIWARE Keyrock
+The following prerequisites are required:
+
+- OpenJDK 11
+- Maven 3.6.3 or higher
+- docker 20.10.12 or higher
+- docker-compose 1.29.2 or higher
 
+FIWARE KeyRock is a component that can handle Identity Management, and it is fully optional to use it with CoatRack. By default, Github is used.
+
+Based on this, in order to make CoatRack work with the FIWARE Keyrock, you will need to:
+
+- First, push and deploy the containers of keyrock via docker (as the Keyrock itself needs to be running in your local machine). To do this, just follow
+the tutorial in https://fiware-idm.readthedocs.io/en/latest/getting_started/index.html;
+- Please check that Keyrock is running correctly in your local machine (to do this, please access localhost:3000 which is the default);
+- If everything is running well, then we need to start by creating an application in Keyrock, to do this:
+  - Login or Create a user account (you can use the admin@test.com, which is the default admin account);
+  ![KeyRock User Login](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockLandingPage.png)
+  - After login, go to the application and click in the register button;
+  ![KeyRock register application](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockRegisterApplication.png)
+  - In the register form, fill up all the fields, but please fill up the url and callback url as seen in the figure below (these will be important as this values will later be in the configuration file in CoatRack);
+  ![KeyRock application form](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockApplicationFormFilled.png)
+  - With your application already created (Attention: only the URL and Callback URL need to be the same, so for example you can give the applcation name as you wish), the next step will be to add this application to the CoatRack config file (please use the application-keyrock.yml).
+  In order to do this configuration, you will need the Oauth2 credentials, so go to the previously created application and click in OAuth2 Credentials, and you should see a ClientId and ClientSecret as shown in the screenshot;
+  ![KeyRock Oauth2 Credentials](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockOauth2Credentials.png)
+  - After the Oauth2 credentials were obtained, please copy them and replace in the application-keyrock.yml, as marked in the figure below;
+  ![KeyRock Oauth2 Credentials](./spring-boot/admin/src/main/resources/static/images/keyrock/KeyRockConfig.png)
+ 
+- After all of this (creating the application in KeyRock and added its config to CoatRack), you can now build and run CoatRack, this could be done in 2 ways:
+  - Via the CLI, by replacing the `-Dspring-boot.run.profiles=github`, with the `-Dspring-boot.run.profiles=keyrock`; 
```

Comment:
Here it would be easier for the reader to put the full command, otherwise they have to search in other parts of the README for the command they need to adjust.